sqlitecluster/SQLiteNode.cpp

```diff
@@ -2716,3 +2719,10 @@ SQLiteNodeState SQLiteNode::stateFromName(const string& name) {
         return it->second;
     }
 }
+
+void SQLiteNode::kill() {
+    for (SQLitePeer* peer : _peerList) {
+        SWARN("Killing peer: " << peer->name);
+        peer->reset();
+    }
+}
```

Comment:
Would prefer to use more informative and less violent naming 

```suggestion
void SQLiteNode::resetPeerConnections() {
    for (SQLitePeer* peer : _peerList) {
        SWARN("Resetting peer connection to " << peer->name);
        peer->reset();
    }
}
```